authentication-saml/src/main/java/com/synopsys/integration/alert/authentication/saml/validator/SAMLFileUploadValidator.java

```diff
@@ -17,10 +18,9 @@
 import javax.xml.parsers.DocumentBuilder;
 import javax.xml.parsers.DocumentBuilderFactory;
 import javax.xml.parsers.ParserConfigurationException;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.IOException;
-import java.io.InputStream;
+import java.io.*;
```

Comment:
I believe we should be avoiding import * from a package.

Reply:
Addressed!